config.yaml

```diff
@@ -62,10 +68,41 @@ menu:
           name: Publishing
           parent: yaml
           weight: 4
+        - identifier: yaml-notification
+          name: Notification
+          parent: yaml
+          weight: 5
+        - identifier: yaml-running-builds
+          name: Running builds
+          parent: yaml
+          weight: 6
         - identifier: integrations
           name: Integrations
           parent: yaml
+          weight: 7
+        - identifier: knowledge-base
+          name: Advanced configuration
+          weight: 8
+        - identifier: knowledge-git
+          name: Git
+          weight: 2
+          parent: knowledge-base
+        - identifier: knowledge-codemagic
+          name: Codemagic
+          weight: 1
+          parent: knowledge-base
+        - identifier: knowledge-firebase
+          name: Firebase
+          weight: 3
+          parent: knowledge-base
+        - identifier: knowledge-publishing
+          name: Publishing
```

Comment:
I'm not entirely convinced it's a good idea to duplicate subcategory names under Configuration and Advanced configuration. Perhaps there's a better way to make the distinction between which publishing articles should be in the two parent categories. One idea for Configuration > Publishing would be to keep this for built-in features and/or stores, and everything else in Advanced configuration. Or make the subcategory in Advanced configuration about uploading artifacts and change the name accordingly.

Reply:
So far, the 'Publishin' section in 'Advanced Configuration' only has 2 items so I would suggest we simply move them to 'Advanced configuration > Various'

Reply:
Sounds good.